src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
well, introducing a segfault to "fix" a coverity error seems not really a gain 

but we could move it to line 117 to close even if fgets fails which should be safe (fp then has a valid file pointer that one can close)